routeros/resource_interface_ethernet.go

```diff
@@ -341,12 +365,52 @@ func ResourceInterfaceEthernet() *schema.Resource {
 			Computed:    true,
 			Description: "Total count of transmitted pause frames.",
 		},
+		"tx_rx_64": {
+			Type:        schema.TypeInt,
+			Computed:    true,
+			Description: "Total count of transmitted and received 64 byte frames",
+		},
+		"tx_rx_65_127": {
+			Type:        schema.TypeInt,
+			Computed:    true,
+			Description: "Total count of transmitted and received 64 to 127 byte frames",
+		},
+		"tx_rx_128_255": {
+			Type:        schema.TypeInt,
+			Computed:    true,
+			Description: "Total count of transmitted and received 128 to 255 byte frames",
+		},
+		"tx_rx_256_511": {
+			Type:        schema.TypeInt,
+			Computed:    true,
+			Description: "Total count of transmitted and received 256 to 511 byte frames",
+		},
+		"tx_rx_512_1023": {
+			Type:        schema.TypeInt,
+			Computed:    true,
+			Description: "Total count of transmitted and received 512 to 1024 byte frames",
+		},
+		"tx_rx_1024_max": {
+			Type:        schema.TypeInt,
+			Computed:    true,
+			Description: "Total count of transmitted and received 1024 or above byte frames",
+		},
+		"tx_underrun": {
+			Type:        schema.TypeInt,
+			Computed:    true,
+			Description: "Total count of transmitted underrun packets",
+		},
+		"tx_unicast": {
+			Type:        schema.TypeInt,
+			Computed:    true,
+			Description: "Total count of transmitted unicast frames.",
+		},
 	}
 
 	return &schema.Resource{
 		CreateContext: UpdateOnlyDeviceCreate(resSchema),
-		ReadContext:   DefaultRead(resSchema),
-		UpdateContext: DefaultUpdate(resSchema),
+		ReadContext:   UpdateOnlyDeviceRead(resSchema),
+		UpdateContext: UpdateOnlyDeviceUpdate(resSchema),
 		DeleteContext: NoOpDelete,
```

Comment:
I seem to have missed that line earlier. I suggest using the DefaultSystemDelete function to notify the user that system resources cannot be deleted.

Reply:
Yep, the delete is fixed.